addons/mail/static/src/core/common/chat_window_model.js

```diff
@@ -79,7 +79,9 @@ export class ChatWindow extends Record {
                 this.notifyState();
             }
         }
-        this.focus();
+        if (!this.store.chatHub.hasComposerFocused) {
+            this.focus();
+        }
```

Comment:
This is probably not the correct fix.
When opening a new chat window manually, the focus should be changed.
What we need is to not call focus depending on a param. Automatically open chat window should never focus it.

Reply:
> When opening a new chat window manually, the focus should be changed.

This PR doesn't change this behavior since the composer will lose the focus when you open a new chat window.